윤건수/baekjoon/G4_1753/Main.java

```diff
@@ -0,0 +1,87 @@
+package baekjoon.G4_1753;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.InputStreamReader;
+import java.io.OutputStreamWriter;
+import java.util.ArrayList;
+import java.util.PriorityQueue;
+import java.util.StringTokenizer;
+
+public class Main {
+
+    public static void main(String[] args) throws Exception {
+
+        int infinite = 999999999;
```

Comment:
```suggestion
        int final INF = Integer.MAX_VALUE;
```

정의되어 있는 상수 Integer.MAX_VALUE 사용은 어떠신가요?